services/static-webserver/client/source/class/osparc/desktop/WorkbenchView.js

```diff
@@ -747,6 +749,20 @@ qx.Class.define("osparc.desktop.WorkbenchView", {
       }
     },
 
+    listenToNoMoreCreditsEvents: function() {
+      const socket = osparc.wrapper.WebSocket.getInstance();
+
+      const slotName = "serviceNoMoreCredits";
+      if (!socket.slotExists(slotName)) {
+        socket.on(slotName, noMoreCredits => {
+          const nodeId = noMoreCredits["node_id"];
+
+          const workbench = this.getStudy().getWorkbench();
```

Comment:
could we emit a notification for the user here?

Reply:
You would need to use something like this
`message = this.tr(`Out of Disk Space on "Service Filesystem" for ${nodeName}<br />The volume Service Filesystem has only ${freeSpace} disk space remaining. You can free up disk space by removing unused files in your service. Alternatively, you can run your service with a pricing plan that supports your storage requirements.`);
            osparc.FlashMessenger.getInstance().logAs(message, "ERROR");`

Reply:
Managed to add this one